code/_onclick/ai.dm

```diff
@@ -63,6 +66,10 @@
 		waypoint_mode = 0
 		return
 
+	if(controlled_mech) //Are we piloting a mech? Placed here so the modifiers are not overridden.
```

Comment:
call()() is incredibly horrendously slow.
we can't have proper variables containing functions in byond riprip